src/main/scala/com/codacy/client/bitbucket/service/PullRequestServices.scala

```diff
@@ -28,6 +28,16 @@ class PullRequestServices(client: BitbucketClient) {
     client.executePaginated(Request(url, classOf[Seq[SimpleCommit]]))
   }
 
+  /*
+   * Gets the list of activities of a pull request
+   *
+   */
+  def getPullRequestActivity(owner: String, repository: String, prId: Long): RequestResponse[Seq[JsObject]] = {
```

Comment:
We would prefer to return a meaningful type instead of a JsObject. Consider creating a new model (you can see an example at https://github.com/codacy/bitbucket-scala-client/blob/master/src/main/scala/com/codacy/client/bitbucket/PullRequest.scala for instance).